indexer/scripts/elastic-conf.py

```diff
@@ -144,6 +154,34 @@ def create_initial_index(self, es: Elasticsearch) -> Any:
                 logger.error("Failed to create Index. Response:%s", response)
             return acknowledged
 
+    def create_slm_policy(self, es: Elasticsearch) -> Any:
+        policy_id = "bi_weekly_slm"
+        repository = self.es_snapshot_repo
+        json_data = self.load_slm_policy_template()
```

Comment:
If we define:
```python
CURRENT_POLICY_ID = "bi_weekly_snapshots"
```
at the top of this file, then:
```suggestion
        repository = self.es_snapshot_repo
        json_data = self.load_slm_policy_template(CURRENT_POLICY_ID)
```